### PR TITLE
Hotfix deprecate nf validate plugin

### PR DESCRIPTION
**Pull request by Jorisvansteenbrugge**
Small change to the nf-core template to replace the now deprecated plugin/nf-validate (used in `validateParamters`) with the successor plugin/nf-scheme.

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -17,12 +17,12 @@ manifest {
...
 // Nextflow plugins
 plugins {
-    id 'nf-validation' // Validation of pipeline parameters and creation of an input channel from a sample sheet
+    id 'nf-schema@2.0.0' // Validation of pipeline parameters and creation of an input channel from a sample sheet
 }
 
 // Global default params, used in configs
 params {
-    genome = '/hpc/diaggen/data/databases/ref_genomes/Homo_sapiens.GRCh37.GATK.illumina/Homo_sapiens.GRCh37.GATK.illumina.fasta'
+    //genome = '/hpc/diaggen/data/databases/ref_genomes/Homo_sapiens.GRCh37.GATK.illumina/Homo_sapiens.GRCh37.GATK.illumina.fasta'
```
Comment (rernst) [inline nextflow.config]: This is just an example, no need to comment out?

File: nextflow.config [hunk 2]
```diff
@@ -39,7 +39,7 @@ params {
 process.shell = ['/bin/bash', '-euo', 'pipefail']
 
 // Load base.config
-includeConfig 'conf/base.config'
+//includeConfig 'conf/base.config'
```
Comment (rernst) [inline nextflow.config]: This config is required to use the default module configurations.

**Review by rernst — CHANGES_REQUESTED** [review]
Revert the two comments? Change to nf-schema is approved.

**Comment by Jorisvansteenbrugge** [comment]
@rernst I reverted the two comments.

**Review by rernst — APPROVED** [review]

**Pull request merged**